draft-ietf-webtrans-http3.md

```diff
@@ -598,6 +602,33 @@ streams.  A QUIC stream's data limit controls the amount of data that can be
 sent on that stream, WebTransport or otherwise (see {{Section 4
 of !RFC9000}}).
 
+## Negotiating the Use of Flow Control {#flow-control-negotiate}
+
+The use of flow control is optional and can be negotiated.  A WebTransport
```

Comment:
> The use of flow control is optional and can be negotiated.

This is a little soft.  It's required when > 1 session, and can't really be negotiated.